src/main/java/com/robosh/ejournal/entity/Teacher.java

```diff
@@ -0,0 +1,27 @@
+package com.robosh.ejournal.entity;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import javax.persistence.Entity;
+
```

Comment:
remove this empty string. Always use ctrl+alt+L to apply code style and ctrl+alt+O to remove redundant  imports